rdkit/Chem/SaltRemover.py

```diff
@@ -33,20 +33,61 @@
 
 import os
 import re
+from rdkit.Chem.rdmolfiles import SDMolSupplier
 
 from rdkit import Chem
 from rdkit import RDConfig
 from contextlib import closing
 
+class Format:
+  Smarts, Mol, Smiles = range(3)
```

Comment:
Is that not redundant? Having a label which is only the constant in lowercase... In which context would the label be used?